doc/api/http.markdown

```diff
@@ -548,6 +548,11 @@ parameter is 511 (not 512).
 This function is asynchronous. The last parameter `callback` will be added as
 a listener for the `'listening'` event.  See also [`net.Server.listen(port)`][].
 
+### server.listening
+
+Returns a Boolean indicating whether or not the server is listening for
```

Comment:
`Returns` implies this is a function returning a value. Instead this is a property of an object. Maybe just remove `Returns`? Also in `doc/api/net.markdown`.